From: template <nomail@local>
Date: Fri, 6 Sep 2019 08:29:50 +0000
Subject: wifi_no_meshfwding_by_default

diff --git a/modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js b/modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js
index 748cb6254f4aeb67300a2e1130aae894717bb455..edcf7552132fc2cd15565c9d5d4bd56737d807cb 100644
--- a/modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js
+++ b/modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js
@@ -943,7 +943,7 @@ return view.extend({
 
 				o = ss.taboption('advanced', form.Flag, 'mesh_fwding', _('Forward mesh peer traffic'));
 				o.rmempty = false;
-				o.default = '1';
+				o.default = '0';
 				o.depends('mode', 'mesh');
 
 				o = ss.taboption('advanced', form.Value, 'mesh_rssi_threshold', _('RSSI threshold for joining'), _('0 = not using RSSI threshold, 1 = do not change driver default'));
