From: template <nomail@local>
Date: Fri, 6 Sep 2019 08:29:50 +0000
Subject: wifi_nolegacy_rates_by_default

diff --git a/modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js b/modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js
index 5d4b3cd70c2378809f77d0b1bae4c15ab17f6785..c2bbf2a3f18bc286b5fbec1397f63cd84294c278 100644
--- a/modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js
+++ b/modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js
@@ -890,7 +890,7 @@ return view.extend({
 					o.wifiNetwork = radioNet;
 
 					o = ss.taboption('advanced', form.Flag, 'legacy_rates', _('Allow legacy 802.11b rates'));
-					o.default = o.enabled;
+					o.default = o.disabled;
 
 					o = ss.taboption('advanced', form.Value, 'distance', _('Distance Optimization'), _('Distance to farthest network member in meters.'));
 					o.datatype = 'range(0,114750)';
