From: Sven Roederer <freifunk@it-solutions.geroedel.de>
Date: Sat, 6 Jul 2019 14:31:15 +0200
Subject: luci-base: update dependency to luci-lib-base

To satisfy the dependencies.

Signed-off-by: Sven Roederer <freifunk@it-solutions.geroedel.de>

diff --git a/modules/luci-base/Makefile b/modules/luci-base/Makefile
index 5019646611cdeeccba071198931c4622dcb753d3..6cb2b640925adbd14d90274da4ff547a265ec7f8 100644
--- a/modules/luci-base/Makefile
+++ b/modules/luci-base/Makefile
@@ -12,7 +12,7 @@ LUCI_TYPE:=mod
 LUCI_BASENAME:=base
 
 LUCI_TITLE:=LuCI core libraries
-LUCI_DEPENDS:=+lua +luci-lib-nixio +luci-lib-ip +rpcd +libubus-lua +luci-lib-jsonc +liblucihttp-lua +rpcd-mod-file +rpcd-mod-luci +cgi-io
+LUCI_DEPENDS:=+lua +luci-lib-nixio +luci-lib-ip +rpcd +libubus-lua +luci-lib-jsonc +liblucihttp-lua +luci-lib-base +rpcd-mod-file +rpcd-mod-luci +cgi-io
 
 PKG_LICENSE:=MIT
 
