From: Sven Roederer <freifunk@it-solutions.geroedel.de>
Date: Sun, 21 Jun 2020 00:31:00 +0200
Subject: add compatibility-wrapper for util.pcdata / striptags

diff --git a/libs/luci-lib-base/luasrc/util.lua b/libs/luci-lib-base/luasrc/util.lua
index f3aa52603f8d7d3960951feeacd4a727727c5d81..1ac0d07cc47c7aa7b732a7f85f522f5b68de20db 100644
--- a/libs/luci-lib-base/luasrc/util.lua
+++ b/libs/luci-lib-base/luasrc/util.lua
@@ -159,6 +159,13 @@ end
 -- String and data manipulation routines
 --
 
+-- compatibility wrapper for xml.pcdata
+function pcdata(value)
+	local xml = require "luci.xml"
+
+	return xml.pcdata(value)
+end
+
 function urlencode(value)
 	if value ~= nil then
 		local str = tostring(value)
@@ -178,6 +185,13 @@ function urldecode(value, decode_plus)
 	return nil
 end
 
+-- compatibility wrapper for xml.striptags
+function striptags(value)
+	local xml = require "luci.xml"
+
+	return xml.striptags(value)
+end
+
 function shellquote(value)
 	return string.format("'%s'", string.gsub(value or "", "'", "'\\''"))
 end
