From: Jakob (XDjackieXD) Riepler <jakob.riepler@chaosfield.at>
Date: Sat, 27 Jun 2020 01:13:37 +0200
Subject: freifunk-common: make olsr status available again for unauthed users

diff --git a/modules/luci-mod-freifunk/luasrc/controller/freifunk/freifunk.lua b/modules/luci-mod-freifunk/luasrc/controller/freifunk/freifunk.lua
index e04cfaf7dfcbcdfa357986a7247310eb2366a360..2e037f59eab39faf6e72be2d596f5e87a6d48cfa 100644
--- a/modules/luci-mod-freifunk/luasrc/controller/freifunk/freifunk.lua
+++ b/modules/luci-mod-freifunk/luasrc/controller/freifunk/freifunk.lua
@@ -52,6 +52,7 @@ function index()
 	page = assign({"freifunk", "olsr"}, {"admin", "status", "olsr"}, _("OLSR"), 30)
 	page.setuser = false
 	page.setgroup = false
+	page.acl_depends = {}
 
 	if nixio.fs.access("/etc/config/luci_statistics") then
 		assign({"freifunk", "graph"}, {"admin", "statistics", "graph"}, _("Statistics"), 40)
