From: template <nomail@local>
Date: Fri, 6 Sep 2019 08:29:50 +0000
Subject: imagebuilder-custom-postinst-script

diff --git a/target/imagebuilder/files/Makefile b/target/imagebuilder/files/Makefile
index 78ec7e2e94a94ebe10d374770b16330ee4014ad8..67ec0e9dbcd5cf3245afe0d37e4177060a4c02b9 100644
--- a/target/imagebuilder/files/Makefile
+++ b/target/imagebuilder/files/Makefile
@@ -143,6 +143,9 @@ _call_image: staging_dir/host/.prereq-build
 	$(MAKE) package_reload
 	$(MAKE) package_install
 	$(MAKE) -s prepare_rootfs
+ifneq ($(CUSTOM_POSTINST_SCRIPT),)
+	(cd "$(TARGET_DIR)" ; bash $(CUSTOM_POSTINST_SCRIPT))
+endif
 	$(MAKE) -s build_image
 	$(MAKE) -s json_overview_image_info
 	$(MAKE) -s checksum
