From: Sven Roederer <devel-sven@geroedel.de>
Date: Fri, 16 Jul 2021 02:16:51 +0200
Subject: ramips: add EdgePoint R6 (as copy of EdgeRouter X SFP)

diff --git a/target/linux/ramips/dts/mt7621_ubnt_edgepoint-r6.dts b/target/linux/ramips/dts/mt7621_ubnt_edgepoint-r6.dts
new file mode 100644
index 0000000000000000000000000000000000000000..de2d11f9bc1f83e69e872089a7bd5a4f6c46570e
--- /dev/null
+++ b/target/linux/ramips/dts/mt7621_ubnt_edgepoint-r6.dts
@@ -0,0 +1,73 @@
+#include "mt7621_ubnt_edgerouter-x.dtsi"
+
+/ {
+	model = "Ubiquiti EdgePoint R6";
+	compatible = "ubnt,edgepoint-r6", "mediatek,mt7621-soc";
+
+	sfp_eth5: sfp_eth5 {
+		compatible = "sff,sfp";
+		i2c-bus = <&i2c>;
+		mod-def0-gpio = <&expander0 5 GPIO_ACTIVE_LOW>;
+		maximum-power-milliwatt = <1000>;
+	};
+};
+
+&i2c {
+	status = "okay";
+
+	/*
+	 * PCA9655 GPIO expander
+	 *  0-POE power port eth0
+	 *  1-POE power port eth1
+	 *  2-POE power port eth2
+	 *  3-POE power port eth3
+	 *  4-POE power port eth4
+	 *  5-SFP_MOD_DEF0#
+	 *  6-
+	 *  7-
+	 *  8-Pull up to VCC
+	 *  9-Pull down to GND
+	 * 10-Pull down to GND
+	 * 11-Pull down to GND
+	 * 12-Pull down to GND
+	 * 13-Pull down to GND
+	 * 14-Pull down to GND
+	 * 15-Pull down to GND
+	 */
+	expander0: pca9555@25 {
+		compatible = "nxp,pca9555";
+		interrupt-parent = <&gpio>;
+		interrupts = <8 IRQ_TYPE_EDGE_FALLING>;
+		gpio-controller;
+		#gpio-cells = <2>;
+		reg = <0x25>;
+	};
+};
+
+&gpio {
+	sfp_i2c_clk_gate {
+		gpio-hog;
+		gpios = <7 GPIO_ACTIVE_LOW>;
+		output-high;
+	};
+};
+
+&mdio {
+	ephy7: ethernet-phy@7 {
+		reg = <7>;
+		sfp = <&sfp_eth5>;
+	};
+};
+
+&switch0 {
+	ports {
+		port@5 {
+			reg = <5>;
+			label = "eth5";
+			phy-handle = <&ephy7>;
+			phy-mode = "rgmii-rxid";
+			mtd-mac-address = <&factory 0x22>;
+			mtd-mac-address-increment = <5>;
+		};
+	};
+};
diff --git a/target/linux/ramips/image/mt7621.mk b/target/linux/ramips/image/mt7621.mk
index a3bc14d59d9923aac77be2f76a2289c05282259e..d76d4b06e85549d93005c5dab16ecd2e4a96fab9 100644
--- a/target/linux/ramips/image/mt7621.mk
+++ b/target/linux/ramips/image/mt7621.mk
@@ -1333,6 +1333,14 @@ define Device/ubnt_edgerouter-x-sfp
 endef
 TARGET_DEVICES += ubnt_edgerouter-x-sfp
 
+define Device/ubnt_edgepoint-r6
+  $(Device/ubnt_edgerouter_common)
+  DEVICE_MODEL := EdgePoint R6
+  DEVICE_PACKAGES += kmod-i2c-algo-pca kmod-gpio-pca953x kmod-sfp
+  SUPPORTED_DEVICES += ubiquiti,edgerouterx-sfp
+endef
+TARGET_DEVICES += ubnt_edgepoint-r6
+
 define Device/ubnt_unifi-6-lite
   $(Device/dsa-migration)
   DEVICE_VENDOR := Ubiquiti
diff --git a/target/linux/ramips/mt7621/base-files/etc/board.d/02_network b/target/linux/ramips/mt7621/base-files/etc/board.d/02_network
index 4b44669b8496af47f6d9391fc0e0bdd2ed2504a6..4d3f67ad82531f0886b1afe3ecfaf8b175bad68c 100755
--- a/target/linux/ramips/mt7621/base-files/etc/board.d/02_network
+++ b/target/linux/ramips/mt7621/base-files/etc/board.d/02_network
@@ -65,6 +65,7 @@ ramips_setup_interfaces()
 	ubnt,edgerouter-x)
 		ucidef_set_interfaces_lan_wan "eth1 eth2 eth3 eth4" "eth0"
 		;;
+	ubnt,edgepoint-r6|\
 	ubnt,edgerouter-x-sfp)
 		ucidef_set_interfaces_lan_wan "eth1 eth2 eth3 eth4 eth5" "eth0"
 		;;
diff --git a/target/linux/ramips/mt7621/base-files/etc/board.d/03_gpio_switches b/target/linux/ramips/mt7621/base-files/etc/board.d/03_gpio_switches
index 8ef815634f0824b3279e4fe13cea46158e16dd8c..8d907120eec14eb6be48b1476d56262871bac04d 100755
--- a/target/linux/ramips/mt7621/base-files/etc/board.d/03_gpio_switches
+++ b/target/linux/ramips/mt7621/base-files/etc/board.d/03_gpio_switches
@@ -16,6 +16,7 @@ telco-electronics,x1)
 ubnt,edgerouter-x)
 	ucidef_add_gpio_switch "poe_passthrough" "PoE Passthrough" "480"
 	;;
+ubnt,edgepoint-r6|\
 ubnt,edgerouter-x-sfp)
 	ucidef_add_gpio_switch "poe_power_port0" "PoE Power Port0" "400"
 	ucidef_add_gpio_switch "poe_power_port1" "PoE Power Port1" "401"
diff --git a/target/linux/ramips/mt7621/base-files/lib/preinit/07_mt7621_bringup_dsa_master b/target/linux/ramips/mt7621/base-files/lib/preinit/07_mt7621_bringup_dsa_master
index 0f4660d242b5bbf1c559518e51f7699029ce6d36..42911c4e03bc36cfa4e15541adf3fb6e3b05658d 100644
--- a/target/linux/ramips/mt7621/base-files/lib/preinit/07_mt7621_bringup_dsa_master
+++ b/target/linux/ramips/mt7621/base-files/lib/preinit/07_mt7621_bringup_dsa_master
@@ -5,6 +5,7 @@ mt7621_bringup_dsa_master() {
     local masterif
 
     case "$board" in
+    ubnt,edgepoint-r6|\
     ubnt,edgerouter-x|\
     ubnt,edgerouter-x-sfp)
         masterif="dsa"
diff --git a/target/linux/ramips/mt7621/base-files/lib/upgrade/platform.sh b/target/linux/ramips/mt7621/base-files/lib/upgrade/platform.sh
index c2ad4d3ed63ebee9207e3323c2df3247b7fbd615..dc8b7abdd2023f762cff2923c28d7e1e62f86167 100755
--- a/target/linux/ramips/mt7621/base-files/lib/upgrade/platform.sh
+++ b/target/linux/ramips/mt7621/base-files/lib/upgrade/platform.sh
@@ -90,6 +90,7 @@ platform_do_upgrade() {
 		iodata_mstc_upgrade_prepare "0x1fe75"
 		nand_do_upgrade "$1"
 		;;
+	ubnt,edgepoint-r6|\
 	ubnt,edgerouter-x|\
 	ubnt,edgerouter-x-sfp)
 		platform_upgrade_ubnt_erx "$1"
