From: Sven Roederer <devel-sven@geroedel.de>
Date: Sat, 24 Apr 2021 15:59:14 +0200
Subject: SDK: filter otu build patts from config.build

diff --git a/target/sdk/convert-config.pl b/target/sdk/convert-config.pl
index f73744af099d2966b1ecacca35483c72b90ec7c6..f6bc831d3a311ee420648fab50aa89ce19fbef28 100755
--- a/target/sdk/convert-config.pl
+++ b/target/sdk/convert-config.pl
@@ -9,7 +9,14 @@ while (<>) {
 	chomp;
 	next if /^CONFIG_SIGNED_PACKAGES/;
 
-	if (/^CONFIG_([^=]+)=(.*)$/) {
+	if (/^CONFIG_((BINARY)|(DOWNLOAD))_FOLDER=(.*)$/) {
+		# We don't want to preserve the build setting of
+		# BINARY_FOLDER and DOWNLOAD_FOLDER.
+		$var = "$1_FOLDER";
+		$val = '""';
+		$type = "string";
+#		warn "DEBUG: type: $type found for symbol CONFIG_$var=$val\n";
+	} elsif (/^CONFIG_([^=]+)=(.*)$/) {
 		$var = $1;
 		$val = $2;
 
