From: adminuser <adminuser@cvt.bln.roederer.dhs.org>
Date: Sun, 27 Sep 2020 03:07:46 +0200
Subject: kernel log_buffer size = 64k (was 128k)

diff --git a/target/linux/ath79/tiny/config-default b/target/linux/ath79/tiny/config-default
index f686d284f67389ad361cb36c72b6f4923ef95493..b8e94ebe4edf2d933f91ba0d521786284aeadda7 100644
--- a/target/linux/ath79/tiny/config-default
+++ b/target/linux/ath79/tiny/config-default
@@ -17,6 +17,7 @@ CONFIG_REGULATOR_FIXED_VOLTAGE=y
 # CONFIG_HAVE_DEBUG_STACKOVERFLOW is not set
 # CONFIG_HAVE_IDE is not set
 CONFIG_BLOCK=y
+CONFIG_LOG_BUF_SHIFT=16
 # CONFIG_HAVE_KVM is not set
 # CONFIG_NVMEM is not set
 # CONFIG_AIO is not set
