From: adminuser <adminuser@cvt.bln.roederer.dhs.org>
Date: Sun, 27 Sep 2020 03:05:42 +0200
Subject: kernel: ath79-tine: optimize for size

diff --git a/target/linux/ath79/tiny/config-default b/target/linux/ath79/tiny/config-default
index f686d284f67389ad361cb36c72b6f4923ef95493..8c7ab419a416efe9a0fe9060700f0c11086ed27a 100644
--- a/target/linux/ath79/tiny/config-default
+++ b/target/linux/ath79/tiny/config-default
@@ -17,6 +17,8 @@ CONFIG_REGULATOR_FIXED_VOLTAGE=y
 # CONFIG_HAVE_DEBUG_STACKOVERFLOW is not set
 # CONFIG_HAVE_IDE is not set
 CONFIG_BLOCK=y
+# CONFIG_CC_OPTIMIZE_FOR_PERFORMANCE is not set
+CONFIG_CC_OPTIMIZE_FOR_SIZE=y
 # CONFIG_HAVE_KVM is not set
 # CONFIG_NVMEM is not set
 # CONFIG_AIO is not set
