From: adminuser <adminuser@cvt.bln.roederer.dhs.org>
Date: Sun, 27 Sep 2020 02:45:53 +0200
Subject: kernel: ath79-tiny: reactivete some kernel feature, as still not bootup-rootfs

diff --git a/target/linux/ath79/tiny/config-default b/target/linux/ath79/tiny/config-default
index 5dcdf30af4720d324cc2bb6bbdaa888bdebb60d5..dada97dccba968ca278dd66eda064c4028b0cea5 100644
--- a/target/linux/ath79/tiny/config-default
+++ b/target/linux/ath79/tiny/config-default
@@ -12,17 +12,12 @@ CONFIG_PHYLINK=y
 # CONFIG_PHY_AR7200_USB is not set
 CONFIG_REGULATOR=y
 CONFIG_REGULATOR_FIXED_VOLTAGE=y
-# CONFIG_USB_SUPPORT is not set
 # CONFIG_HAVE_DEBUG_KMEMLEAK is not set
 # CONFIG_HAVE_DEBUG_STACKOVERFLOW is not set
-# CONFIG_HAVE_IDE is not set
 # CONFIG_HAVE_KVM is not set
-# CONFIG_NVMEM is not set
-# CONFIG_AIO is not set
 # CONFIG_DEBUG_FS is not set
 # CONFIG_DEBUG_KERNEL is not set
 # CONFIG_ISDN is not set
 # CONFIG_SND_DRIVERS is not set
-# CONFIG_STAGING is not set
 # CONFIG_VIRTIO_MENU is not set
 
