From: Sven Roederer <devel-sven@geroedel.de>
Date: Wed, 21 Apr 2021 08:33:27 +0200
Subject: build: remove NPROC_MAX; allow externally overwriting NPROC

diff --git a/config/Config-devel.in b/config/Config-devel.in
index 11ddc85b6e5b4721e9d0efcf6458dd72704f248f..126462bfc3a0a12324b3ac38614c4f94638d0115 100644
--- a/config/Config-devel.in
+++ b/config/Config-devel.in
@@ -45,15 +45,6 @@ menuconfig DEVEL
 		  This allows you to symlink build_dir into a scratch location, e.g. a ramdisk,
 		  which does not have enough space to keep a complete build_dir.
 
-	config NPROC_MAX
-		int "max CPUs to be used" if DEVEL
-		default 0
-		help
-		   Forces to use this many CPU in parallel at max.
-		   A value of 0 uses autodetection for max available CPUs. All other values
-		   definie the upper limit.
-		   This is currently relevant for the compression of the imagebuilder and sdk.
-
 	config BUILD_SUFFIX
 		string "Build suffix to append to the target BUILD_DIR variable" if DEVEL
 		default ""
diff --git a/rules.mk b/rules.mk
index 8f41ff5861a046bcad893b9a11874f96e3f43e3e..c52227be588fcf9ee9b96c04cfda5a877a14a8e1 100644
--- a/rules.mk
+++ b/rules.mk
@@ -67,7 +67,7 @@ TARGET_SUFFIX=$(call qstrip,$(CONFIG_TARGET_SUFFIX))
 BUILD_SUFFIX:=$(call qstrip,$(CONFIG_BUILD_SUFFIX))
 SUBDIR:=$(patsubst $(TOPDIR)/%,%,${CURDIR})
 BUILD_SUBDIR:=$(patsubst $(TOPDIR)/%,%,${CURDIR})
-NPROC:=$(shell sysctl -n hw.ncpu 2>/dev/null || nproc)
+NPROC?=$(shell sysctl -n hw.ncpu 2>/dev/null || nproc)
 export SHELL:=/usr/bin/env bash
 
 IS_PACKAGE_BUILD := $(if $(filter package/%,$(BUILD_SUBDIR)),1)
diff --git a/target/imagebuilder/Makefile b/target/imagebuilder/Makefile
index 2c6b42cf8950d3768e86c4758a3c270b879d2ed8..3e69ae4a13e7bcc3ab60b48dd11a0261483fce49 100644
--- a/target/imagebuilder/Makefile
+++ b/target/imagebuilder/Makefile
@@ -107,7 +107,7 @@ endif
 	$(CP) $(TOPDIR)/staging_dir/host/lib/libfakeroot* $(PKG_BUILD_DIR)/staging_dir/host/lib
 	STRIP=$(STAGING_DIR_HOST)/bin/sstrip $(SCRIPT_DIR)/rstrip.sh $(PKG_BUILD_DIR)/staging_dir/host/bin/
 	(cd $(BUILD_DIR); \
-		tar -I '$(STAGING_DIR_HOST)/bin/xz -7e -T$(if $(CONFIG_NPROC_MAX),$(CONFIG_NPROC_MAX),$(if $(filter 1,$(NPROC)),2,0))' -cf $@ $(IB_NAME) \
+		tar -I '$(STAGING_DIR_HOST)/bin/xz -7e -T$(if $(filter 1,$(NPROC)),2,0)' -cf $@ $(IB_NAME) \
 		--mtime="$(shell date --date=@$(SOURCE_DATE_EPOCH))"; \
 	)
 
diff --git a/target/sdk/Makefile b/target/sdk/Makefile
index 61918259c7cb231d6f66ccc84060b86a968dd723..d85c17b613e518bffac996932e0fd090511660d6 100644
--- a/target/sdk/Makefile
+++ b/target/sdk/Makefile
@@ -166,7 +166,7 @@ $(BIN_DIR)/$(SDK_NAME).tar.xz: clean
 	find $(SDK_BUILD_DIR) -name CVS | $(XARGS) rm -rf
 	-make -C $(SDK_BUILD_DIR)/scripts/config clean
 	(cd $(BUILD_DIR); \
-		tar -I '$(STAGING_DIR_HOST)/bin/xz -7e -T$(if $(CONFIG_NPROC_MAX),$(CONFIG_NPROC_MAX),$(if $(filter 1,$(NPROC)),2,0))' -cf $@ $(SDK_NAME) \
+		tar -I '$(STAGING_DIR_HOST)/bin/xz -7e -T$(if $(filter 1,$(NPROC)),2,0)' -cf $@ $(SDK_NAME) \
 		--mtime="$(shell date --date=@$(SOURCE_DATE_EPOCH))"; \
 	)
 
