From: adminuser <adminuser@cvt.bln.roederer.dhs.org>
Date: Sun, 27 Sep 2020 03:13:20 +0200
Subject: kernel: ath79-tine: enable CONFIG_BLOCK

diff --git a/target/linux/ath79/tiny/config-default b/target/linux/ath79/tiny/config-default
index 8897591d3165dbf10cac948a4cccdac48018e9ff..3d2e0646f42c445cfa606b51c37bc4ca7c537730 100644
--- a/target/linux/ath79/tiny/config-default
+++ b/target/linux/ath79/tiny/config-default
@@ -15,6 +15,7 @@ CONFIG_REGULATOR_FIXED_VOLTAGE=y
 # CONFIG_CC_OPTIMIZE_FOR_PERFORMANCE is not set
 CONFIG_CC_OPTIMIZE_FOR_SIZE=y
 CONFIG_LOG_BUF_SHIFT=16
+CONFIG_BLOCK=y
 # CONFIG_HAVE_KVM is not set
 # CONFIG_DEBUG_FS is not set
 # CONFIG_DEBUG_KERNEL is not set
