From: Sven Roederer <devel-sven@geroedel.de>
Date: Thu, 22 Apr 2021 00:02:48 +0200
Subject: build: imagebuilder, SDK accept NPROC_OVERRIDE

diff --git a/target/imagebuilder/Makefile b/target/imagebuilder/Makefile
index 3e69ae4a13e7bcc3ab60b48dd11a0261483fce49..6a6b6f6effd5b07a616c53f74fa6d7e56b1d2d2f 100644
--- a/target/imagebuilder/Makefile
+++ b/target/imagebuilder/Makefile
@@ -107,7 +107,7 @@ endif
 	$(CP) $(TOPDIR)/staging_dir/host/lib/libfakeroot* $(PKG_BUILD_DIR)/staging_dir/host/lib
 	STRIP=$(STAGING_DIR_HOST)/bin/sstrip $(SCRIPT_DIR)/rstrip.sh $(PKG_BUILD_DIR)/staging_dir/host/bin/
 	(cd $(BUILD_DIR); \
-		tar -I '$(STAGING_DIR_HOST)/bin/xz -7e -T$(if $(filter 1,$(NPROC)),2,0)' -cf $@ $(IB_NAME) \
+		tar -I '$(STAGING_DIR_HOST)/bin/xz -7e -T$(if $(NPROC_OVERRIDE),$(NPROC_OVERRIDE),$(if $(filter 1,$(NPROC)),2,0))' -cf $@ $(IB_NAME) \
 		--mtime="$(shell date --date=@$(SOURCE_DATE_EPOCH))"; \
 	)
 
diff --git a/target/sdk/Makefile b/target/sdk/Makefile
index d85c17b613e518bffac996932e0fd090511660d6..96ab628f29a90045a8de2433b543e20b1a0e817c 100644
--- a/target/sdk/Makefile
+++ b/target/sdk/Makefile
@@ -166,7 +166,7 @@ $(BIN_DIR)/$(SDK_NAME).tar.xz: clean
 	find $(SDK_BUILD_DIR) -name CVS | $(XARGS) rm -rf
 	-make -C $(SDK_BUILD_DIR)/scripts/config clean
 	(cd $(BUILD_DIR); \
-		tar -I '$(STAGING_DIR_HOST)/bin/xz -7e -T$(if $(filter 1,$(NPROC)),2,0)' -cf $@ $(SDK_NAME) \
+		tar -I '$(STAGING_DIR_HOST)/bin/xz -7e -T$(if $(NPROC_OVERRIDE),$(NPROC_OVERRIDE),$(if $(filter 1,$(NPROC)),2,0))' -cf $@ $(SDK_NAME) \
 		--mtime="$(shell date --date=@$(SOURCE_DATE_EPOCH))"; \
 	)
 
