From: Sven Roederer <devel-sven@geroedel.de>
Date: Sat, 27 Nov 2021 03:57:58 +0100
Subject: linux: add kmod-lzo-rle

diff --git a/package/kernel/linux/modules/lib.mk b/package/kernel/linux/modules/lib.mk
index 9a341932bdeb3c9816b82eeeecc7114f24615fcc..1bc4933a50791c01a51e85235052636576704a8c 100644
--- a/package/kernel/linux/modules/lib.mk
+++ b/package/kernel/linux/modules/lib.mk
@@ -121,6 +121,29 @@ endef
 $(eval $(call KernelPackage,lib-lzo))
 
 
+define KernelPackage/lib-lzo-rle
+  SUBMENU:=$(LIB_MENU)
+  TITLE:=LZO-RLE support
+  DEPENDS:=+kmod-crypto-acompress
+  KCONFIG:= \
+	CONFIG_CRYPTO_LZO \
+	CONFIG_LZO_COMPRESS \
+	CONFIG_LZO_DECOMPRESS
+  HIDDEN:=1
+  FILES:= \
+	$(LINUX_DIR)/crypto/lzo-rle.ko \
+	$(LINUX_DIR)/lib/lzo/lzo_compress.ko \
+	$(LINUX_DIR)/lib/lzo/lzo_decompress.ko
+  AUTOLOAD:=$(call AutoProbe,lzo-rle lzo_compress lzo_decompress)
+endef
+
+define KernelPackage/lib-lzo-rle/description
+ Kernel module for LZO-rle compression/decompression support
+endef
+
+$(eval $(call KernelPackage,lib-lzo-rle))
+
+
 define KernelPackage/lib-zstd
   SUBMENU:=$(LIB_MENU)
   TITLE:=ZSTD support
