From d61163695579706610a429562712e848ea4ba81a Mon Sep 17 00:00:00 2001
From: adminuser <adminuser@cvt.bln.roederer.dhs.org>
Date: Sun, 27 Sep 2020 03:07:46 +0200
Subject: [PATCH 1/2] kernel log_buffer size = 64k (was 128k)

---
 target/linux/ath79/tiny/config-default | 1 +
 1 file changed, 1 insertion(+)

diff --git a/target/linux/ath79/tiny/config-default b/target/linux/ath79/tiny/config-default
index f1f743fc0d..1d4859a275 100644
--- a/target/linux/ath79/tiny/config-default
+++ b/target/linux/ath79/tiny/config-default
@@ -15,6 +15,7 @@ CONFIG_REGULATOR_FIXED_VOLTAGE=y
 # CONFIG_USB_SUPPORT is not set
 # CONFIG_HAVE_DEBUG_KMEMLEAK is not set
 # CONFIG_HAVE_DEBUG_STACKOVERFLOW is not set
+CONFIG_LOG_BUF_SHIFT=16
 CONFIG_BLOCK=y
 # CONFIG_HAVE_IDE is not set
 # CONFIG_HAVE_KVM is not set
-- 
2.17.1

