From: Sven Roederer <devel-sven@geroedel.de>
Date: Thu, 22 Apr 2021 00:02:48 +0200
Subject: build: imagebuilder, SDK accept NPROC_OVERRIDE

diff --git a/target/imagebuilder/Makefile b/target/imagebuilder/Makefile
index ef7fd3f25e856c4173f78ace45a98090f61a0779..e29d1899d46e49ee5899e5dc65805b2d467ed87c 100644
--- a/target/imagebuilder/Makefile
+++ b/target/imagebuilder/Makefile
@@ -107,7 +107,7 @@ endif
 	$(CP) $(TOPDIR)/staging_dir/host/lib/libfakeroot* $(PKG_BUILD_DIR)/staging_dir/host/lib
 	STRIP=$(STAGING_DIR_HOST)/bin/sstrip $(SCRIPT_DIR)/rstrip.sh $(PKG_BUILD_DIR)/staging_dir/host/bin/
 	(cd $(BUILD_DIR); \
-		tar -I '$(STAGING_DIR_HOST)/bin/xz -7e -T$(if $(filter 1,$(NPROC)),2,0)' -cf $@ $(IB_NAME) \
+		tar -I '$(STAGING_DIR_HOST)/bin/xz -7e -T$(if $(NPROC_OVERRIDE),$(NPROC_OVERRIDE),$(if $(filter 1,$(NPROC)),2,0))' -cf $@ $(IB_NAME) \
 		--mtime="$(shell date --date=@$(SOURCE_DATE_EPOCH))"; \
 	)
 
diff --git a/target/sdk/Makefile b/target/sdk/Makefile
index 5330d1495543bdfdb4286d01e7ea7300ce9239dc..e9ddf059a85659040d670573e5e010748c28e2cf 100644
--- a/target/sdk/Makefile
+++ b/target/sdk/Makefile
@@ -165,7 +165,7 @@ $(BIN_DIR)/$(SDK_NAME).tar.xz: clean
 	find $(SDK_BUILD_DIR) -name CVS | $(XARGS) rm -rf
 	-make -C $(SDK_BUILD_DIR)/scripts/config clean
 	(cd $(BUILD_DIR); \
-		tar -I '$(STAGING_DIR_HOST)/bin/xz -7e -T$(if $(filter 1,$(NPROC)),2,0)' -cf $@ $(SDK_NAME) \
+		tar -I '$(STAGING_DIR_HOST)/bin/xz -7e -T$(if $(NPROC_OVERRIDE),$(NPROC_OVERRIDE),$(if $(filter 1,$(NPROC)),2,0))' -cf $@ $(SDK_NAME) \
 		--mtime="$(shell date --date=@$(SOURCE_DATE_EPOCH))"; \
 	)
 
