From: Sven Roederer <devel-sven@geroedel.de>
Date: Sun, 27 Sep 2020 04:16:56 +0200
Subject: kernel: ath79-tiny: readd CONFIG_HAVE_DEBUG_*

diff --git a/target/linux/ath79/tiny/config-default b/target/linux/ath79/tiny/config-default
index 3d2e0646f42c445cfa606b51c37bc4ca7c537730..5b7e93884553bc1da375858295d72a7bd28f0ace 100644
--- a/target/linux/ath79/tiny/config-default
+++ b/target/linux/ath79/tiny/config-default
@@ -12,6 +12,8 @@ CONFIG_PHYLINK=y
 # CONFIG_PHY_AR7200_USB is not set
 CONFIG_REGULATOR=y
 CONFIG_REGULATOR_FIXED_VOLTAGE=y
+# CONFIG_HAVE_DEBUG_KMEMLEAK is not set
+# CONFIG_HAVE_DEBUG_STACKOVERFLOW is not set
 # CONFIG_CC_OPTIMIZE_FOR_PERFORMANCE is not set
 CONFIG_CC_OPTIMIZE_FOR_SIZE=y
 CONFIG_LOG_BUF_SHIFT=16
