From: Sven Roederer <devel-sven@geroedel.de>
Date: Fri, 30 Apr 2021 23:51:59 +0200
Subject: opkg: rollback to "2020-08-31.cf44c2feb606"

I expect this as point where it breaks.

diff --git a/package/system/opkg/Makefile b/package/system/opkg/Makefile
index 6c3682571181fb45211d7faf63c59df8c7868373..05a99bde09cbe0107fe5579cfed63eda67f2a937 100644
--- a/package/system/opkg/Makefile
+++ b/package/system/opkg/Makefile
@@ -14,9 +14,9 @@ PKG_FLAGS:=essential
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://git.openwrt.org/project/opkg-lede.git
-PKG_SOURCE_DATE:=2021-03-14
-PKG_SOURCE_VERSION:=d3a63b327383a2ac1a8f925a58a2f6a7808f5181
-PKG_MIRROR_HASH:=65339e965b3d3d2d49b98e41d774bcedb0f044335c651d306d95cd3e96ca57c3
+PKG_SOURCE_DATE:=2020-08-31
+PKG_SOURCE_VERSION:=cf44c2feb606b60ba3f35392e102c281ef6ea0da
+PKG_MIRROR_HASH:=0d7be9f8f86864e2a9999145210ecaa286fb85cfae131ec502609a6d1e435871
 
 PKG_LICENSE:=GPL-2.0
 PKG_LICENSE_FILES:=COPYING
