From: Sven Roederer <devel-sven@geroedel.de>
Date: Sat, 17 Apr 2021 14:37:12 +0200
Subject: rework packege define

diff --git a/addons/freifunk-berlin-lib-guard/Makefile b/addons/freifunk-berlin-lib-guard/Makefile
index 0ba782ca6ff0017ff8899ce0bbe11487d5f7c0a9..6229d60f956e6b836a3dd13b39761e700396464b 100644
--- a/addons/freifunk-berlin-lib-guard/Makefile
+++ b/addons/freifunk-berlin-lib-guard/Makefile
@@ -1,34 +1,12 @@
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=freifunk-berlin-lib-guard
 PKG_RELEASE:=2
 
-include ../../freifunk-berlin-generic.mk
-
-define Package/freifunk-berlin-lib-guard
-  SECTION:=freifunk-berlin
-  CATEGORY:=freifunk-berlin
-  TITLE:=Freifunk Berlin guard function
+LUCI_TITLE:=Freifunk Berlin guard function
   URL:=http://github.com/freifunk-berlin/packages_berlin
-  PKGARCH:=all
-endef
-
-define Package/freifunk-berlin-lib-guard/description
-  Freifunk Berlin guard function
-endef
 
-define Build/Prepare
-	mkdir -p $(PKG_BUILD_DIR)
-endef
+LUCI_DESCRIPTION:=Freifunk Berlin guard function
 
-define Build/Configure
-endef
-
-define Build/Compile
-endef
-
-define Package/freifunk-berlin-lib-guard/install
-	$(CP) ./files/* $(1)/
-endef
+include ../../freifunk-berlin-generic.mk
 
-$(eval $(call BuildPackage,freifunk-berlin-lib-guard))
+# call BuildPackage - is done via freifunk-berlin-generic.mk --> luci.mk
diff --git a/defaults/freifunk-berlin-dhcp-defaults/Makefile b/defaults/freifunk-berlin-dhcp-defaults/Makefile
index f1232b1a2233ecb1aba5af992e8c1569f7c59827..db5a5755987c8aa4a2bbfa5b7ff17113b949ed22 100644
--- a/defaults/freifunk-berlin-dhcp-defaults/Makefile
+++ b/defaults/freifunk-berlin-dhcp-defaults/Makefile
@@ -1,37 +1,11 @@
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=freifunk-berlin-dhcp-defaults
-
-include ../../freifunk-berlin-generic.mk
-
-define Package/freifunk-berlin-dhcp-defaults
-  SECTION:=freifunk-berlin
-  CATEGORY:=freifunk-berlin
-  TITLE:=Freifunk Berlin dhcp default configuration
+LUCI_TITLE:=Freifunk Berlin dhcp default configuration
   URL:=http://github.com/freifunk-berlin/packages_berlin
   DEPENDS+= +dnsmasq +freifunk-berlin-lib-guard
-  PKGARCH:=all
-endef
-
-define Package/freifunk-berlin-dhcp-defaults/description
-  Freifunk Berlin configuration files for dhcp
-endef
 
-define Build/Prepare
-	mkdir -p $(PKG_BUILD_DIR)
-endef
+LUCI_DESCRIPTION=Freifunk Berlin configuration files for dhcp
 
-define Build/Configure
-endef
-
-define Build/Compile
-endef
-
-define Package/freifunk-berlin-dhcp-defaults/install
-	$(INSTALL_DIR) $(1)/etc/uci-defaults
-	$(CP) ./uci-defaults/* $(1)/etc/uci-defaults
-	$(CP) ./files/* $(1)/
-endef
-
-$(eval $(call BuildPackage,freifunk-berlin-dhcp-defaults))
+include ../../freifunk-berlin-generic.mk
 
+# call BuildPackage - is done via freifunk-berlin-generic.mk --> luci.mk
diff --git a/defaults/freifunk-berlin-firewall-defaults/Makefile b/defaults/freifunk-berlin-firewall-defaults/Makefile
index 3f2ec54cd934b3c4b74e76c25bf169f4f4a990bf..141e32cc99ba0b1b0aacef53e6541178cb730a2c 100644
--- a/defaults/freifunk-berlin-firewall-defaults/Makefile
+++ b/defaults/freifunk-berlin-firewall-defaults/Makefile
@@ -1,35 +1,12 @@
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=freifunk-berlin-firewall-defaults
 
-include ../../freifunk-berlin-generic.mk
-
-define Package/freifunk-berlin-firewall-defaults
-  SECTION:=freifunk-berlin
-  CATEGORY:=freifunk-berlin
-  TITLE:=Freifunk Berlin firewall default configuration
+LUCI_TITLE:=Freifunk Berlin firewall default configuration
   URL:=http://github.com/freifunk-berlin/packages_berlin
   DEPENDS+= +firewall +freifunk-berlin-lib-guard
-  PKGARCH:=all
-endef
-
-define Package/freifunk-berlin-firewall-defaults/description
-  Freifunk Berlin configuration files for firewall
-endef
 
-define Build/Prepare
-	mkdir -p $(PKG_BUILD_DIR)
-endef
+LUCI_DESCRIPTION=Freifunk Berlin configuration files for firewall
 
-define Build/Configure
-endef
-
-define Build/Compile
-endef
-
-define Package/freifunk-berlin-firewall-defaults/install
-	$(INSTALL_DIR) $(1)/etc/uci-defaults
-	$(CP) ./uci-defaults/* $(1)/etc/uci-defaults
-endef
+include ../../freifunk-berlin-generic.mk
 
-$(eval $(call BuildPackage,freifunk-berlin-firewall-defaults))
+# call BuildPackage - is done via freifunk-berlin-generic.mk --> luci.mk
diff --git a/defaults/freifunk-berlin-freifunk-defaults/Makefile b/defaults/freifunk-berlin-freifunk-defaults/Makefile
index 3fcccfa414fad1ba96311ccf4bf58c71c5763ffa..0000e0384b480ae795595042e721cb2edd477b11 100644
--- a/defaults/freifunk-berlin-freifunk-defaults/Makefile
+++ b/defaults/freifunk-berlin-freifunk-defaults/Makefile
@@ -1,34 +1,10 @@
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=freifunk-berlin-freifunk-defaults
-
-include ../../freifunk-berlin-generic.mk
-
-define Package/freifunk-berlin-freifunk-defaults
-  SECTION:=freifunk-berlin
-  CATEGORY:=freifunk-berlin
-  TITLE:=Freifunk Berlin freifunk default configuration
+LUCI_TITLE:=Freifunk Berlin freifunk default configuration
   URL:=http://github.com/freifunk-berlin/packages_berlin
-  PKGARCH:=all
-endef
-
-define Package/freifunk-berlin-freifunk-defaults/description
-  Freifunk Berlin configuration files for freifunk
-endef
 
-define Build/Prepare
-	mkdir -p $(PKG_BUILD_DIR)
-endef
+LUCI_DESCRIPTION=Freifunk Berlin configuration files for freifunk
 
-define Build/Configure
-endef
-
-define Build/Compile
-endef
-
-define Package/freifunk-berlin-freifunk-defaults/install
-	$(INSTALL_DIR) $(1)/etc/uci-defaults
-	$(CP) ./uci-defaults/* $(1)/etc/uci-defaults
-endef
+include ../../freifunk-berlin-generic.mk
 
-$(eval $(call BuildPackage,freifunk-berlin-freifunk-defaults))
+# call BuildPackage - is done via freifunk-berlin-generic.mk --> luci.mk
diff --git a/defaults/freifunk-berlin-network-defaults/Makefile b/defaults/freifunk-berlin-network-defaults/Makefile
index d26c2e8ff4a1e218c13d0cc699a8655e94415906..2ce8ecbaaeb5a9383aefed9093fef80765cf4ecf 100644
--- a/defaults/freifunk-berlin-network-defaults/Makefile
+++ b/defaults/freifunk-berlin-network-defaults/Makefile
@@ -1,43 +1,11 @@
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=freifunk-berlin-network-defaults
-
-include ../../freifunk-berlin-generic.mk
-
-define Package/freifunk-berlin-network-defaults
-  SECTION:=freifunk-berlin
-  CATEGORY:=freifunk-berlin
-  TITLE:=Freifunk Berlin network default configuration
+LUCI_TITLE:=Freifunk Berlin network default configuration
   URL:=http://github.com/freifunk-berlin/packages_berlin
   DEPENDS+= +freifunk-berlin-lib-guard +iwinfo +pingcheck
-  PKGARCH:=all
-endef
-
-define Package/freifunk-berlin-network-defaults/description
-  Freifunk Berlin configuration files for network
-endef
 
-define Build/Prepare
-	mkdir -p $(PKG_BUILD_DIR)
-endef
+LUCI_DESCRIPTION=Freifunk Berlin configuration files for network
 
-define Build/Configure
-endef
-
-define Build/Compile
-endef
-
-define Package/freifunk-berlin-network-defaults/install
-	$(INSTALL_DIR) $(1)/etc/uci-defaults
-	$(CP) ./uci-defaults/* $(1)/etc/uci-defaults
-	$(INSTALL_DIR) $(1)/etc/hotplug.d/iface
-	$(CP) ./hotplug.d/iface/* $(1)/etc/hotplug.d/iface/
-	$(INSTALL_DIR) $(1)/lib/functions
-	$(CP) ./lib/functions/* $(1)/lib/functions/
-	$(INSTALL_DIR) $(1)/etc/pingcheck/online.d
-	$(CP) ./pingcheck/online.d/* $(1)/etc/pingcheck/online.d/
-	$(INSTALL_DIR) $(1)/etc/pingcheck/offline.d
-	$(CP) ./pingcheck/offline.d/* $(1)/etc/pingcheck/offline.d/
-endef
+include ../../freifunk-berlin-generic.mk
 
-$(eval $(call BuildPackage,freifunk-berlin-network-defaults))
+# call BuildPackage - is done via freifunk-berlin-generic.mk --> luci.mk
diff --git a/defaults/freifunk-berlin-olsrd-defaults/Makefile b/defaults/freifunk-berlin-olsrd-defaults/Makefile
index 87f442e58d03e12d52c8a8db80bf8026de937907..9f9ed1773e50aed28ad7c8827e560061dccac09d 100644
--- a/defaults/freifunk-berlin-olsrd-defaults/Makefile
+++ b/defaults/freifunk-berlin-olsrd-defaults/Makefile
@@ -1,35 +1,11 @@
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=freifunk-berlin-olsrd-defaults
-
-include ../../freifunk-berlin-generic.mk
-
-define Package/freifunk-berlin-olsrd-defaults
-  SECTION:=freifunk-berlin
-  CATEGORY:=freifunk-berlin
-  TITLE:=Freifunk Berlin olsrd default configuration
+LUCI_TITLE:=Freifunk Berlin olsrd default configuration
   URL:=http://github.com/freifunk-berlin/packages_berlin
   DEPENDS+= +olsrd +freifunk-berlin-lib-guard
-  PKGARCH:=all
-endef
-
-define Package/freifunk-berlin-olsrd-defaults/description
-  Freifunk Berlin configuration files for olsrd and olsrd6
-endef
 
-define Build/Prepare
-	mkdir -p $(PKG_BUILD_DIR)
-endef
+LUCI_DESCRIPTION=Freifunk Berlin configuration files for olsrd and olsrd6
 
-define Build/Configure
-endef
-
-define Build/Compile
-endef
-
-define Package/freifunk-berlin-olsrd-defaults/install
-	$(INSTALL_DIR) $(1)/etc/uci-defaults
-	$(CP) ./uci-defaults/* $(1)/etc/uci-defaults
-endef
+include ../../freifunk-berlin-generic.mk
 
-$(eval $(call BuildPackage,freifunk-berlin-olsrd-defaults))
+# call BuildPackage - is done via freifunk-berlin-generic.mk --> luci.mk
diff --git a/defaults/freifunk-berlin-openvpn-files/Makefile b/defaults/freifunk-berlin-openvpn-files/Makefile
index 8c8f4cc5ab3fc2a8b8dc339cdb762cd848b4cc3f..7f28da4e1705fccd868a276575c327eb7f1f3c5d 100644
--- a/defaults/freifunk-berlin-openvpn-files/Makefile
+++ b/defaults/freifunk-berlin-openvpn-files/Makefile
@@ -1,41 +1,10 @@
-# $Id: Makefile 2010-02-04 23:25:21Z pg $
-
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=freifunk-berlin-openvpn-files
-
-include ../../freifunk-berlin-generic.mk
-
-define Package/freifunk-berlin-openvpn-files
-  SECTION:=freifunk-berlin
-  CATEGORY:=freifunk-berlin
-  TITLE:=Freifunk Berlin openVPN files (commmon)
+LUCI_TITLE:=Freifunk Berlin openVPN files (commmon)
   URL:=http://github.com/freifunk-berlin/packages_berlin
   DEPENDS+= +openvpn +freifunk-berlin-lib-guard +freifunk-berlin-network-defaults
-  PKGARCH:=all
-endef
-
-define Package/freifunk-berlin-openvpn-files/description
-  Freifunk Berlin openVPN files
-endef
+LUCI_DESCRIPTION=Freifunk Berlin openVPN files
 
-define Build/Prepare
-	mkdir -p $(PKG_BUILD_DIR)
-endef
-
-define Build/Configure
-endef
-
-define Build/Compile
-endef
-
-define Package/freifunk-berlin-openvpn-files/install
-	$(INSTALL_DIR) $(1)/etc/uci-defaults
-	$(CP) ./uci-defaults/* $(1)/etc/uci-defaults
-	$(INSTALL_DIR) $(1)/lib/freifunk
-	$(CP) ./openvpn/ffvpn-up.sh $(1)/lib/freifunk
-	$(INSTALL_DIR) $(1)/etc/hotplug.d/iface
-	$(CP) ./openvpn/60-ffopenvpn $(1)/etc/hotplug.d/iface
-endef
+include ../../freifunk-berlin-generic.mk
 
-$(eval $(call BuildPackage,freifunk-berlin-openvpn-files))
+# call BuildPackage - is done via freifunk-berlin-generic.mk --> luci.mk
diff --git a/defaults/freifunk-berlin-statistics-defaults/Makefile b/defaults/freifunk-berlin-statistics-defaults/Makefile
index 831bfb1103d22cd0426960dc78541e16de9eff36..09496edac5e34d6a9f0b5e588089ccf402362fc6 100644
--- a/defaults/freifunk-berlin-statistics-defaults/Makefile
+++ b/defaults/freifunk-berlin-statistics-defaults/Makefile
@@ -1,35 +1,11 @@
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=freifunk-berlin-statistics-defaults
-
-include ../../freifunk-berlin-generic.mk
-
-define Package/freifunk-berlin-statistics-defaults
-  SECTION:=freifunk-berlin
-  CATEGORY:=freifunk-berlin
-  TITLE:=Freifunk Berlin statistics default configuration
+LUCI_TITLE:=Freifunk Berlin statistics default configuration
   URL:=http://github.com/freifunk-berlin/packages_berlin
   DEPENDS+= +olsrd-mod-txtinfo +freifunk-berlin-lib-guard
-  PKGARCH:=all
-endef
-
-define Package/freifunk-berlin-statistics-defaults/description
-  Freifunk Berlin configuration files for statistics
-endef
 
-define Build/Prepare
-	mkdir -p $(PKG_BUILD_DIR)
-endef
+LUCI_DESCRIPTION=Freifunk Berlin configuration files for statistics
 
-define Build/Configure
-endef
-
-define Build/Compile
-endef
-
-define Package/freifunk-berlin-statistics-defaults/install
-	$(INSTALL_DIR) $(1)/etc/uci-defaults
-	$(CP) ./uci-defaults/* $(1)/etc/uci-defaults
-endef
+include ../../freifunk-berlin-generic.mk
 
-$(eval $(call BuildPackage,freifunk-berlin-statistics-defaults))
+# call BuildPackage - is done via freifunk-berlin-generic.mk --> luci.mk
diff --git a/defaults/freifunk-berlin-system-defaults/Makefile b/defaults/freifunk-berlin-system-defaults/Makefile
index adeb567a53a6764763bc0045e11a71eaac7dcb4b..437a684f435b4f3caeedcd4d04d787ffe02d2ac1 100644
--- a/defaults/freifunk-berlin-system-defaults/Makefile
+++ b/defaults/freifunk-berlin-system-defaults/Makefile
@@ -1,35 +1,10 @@
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=freifunk-berlin-system-defaults
-
-include ../../freifunk-berlin-generic.mk
-
-define Package/freifunk-berlin-system-defaults
-  SECTION:=freifunk-berlin
-  CATEGORY:=freifunk-berlin
-  TITLE:=Freifunk Berlin system default configuration
+LUCI_TITLE:=Freifunk Berlin system default configuration
   URL:=http://github.com/freifunk-berlin/packages_berlin
   DEPENDS+= +freifunk-berlin-lib-guard
-  PKGARCH:=all
-endef
-
-define Package/freifunk-berlin-system-defaults/description
-  Freifunk Berlin configuration files for basic system-files
-endef
+LUCI_DESCRIPTION=Freifunk Berlin configuration files for basic system-files
 
-define Build/Prepare
-	mkdir -p $(PKG_BUILD_DIR)
-endef
-
-define Build/Configure
-endef
-
-define Build/Compile
-endef
-
-define Package/freifunk-berlin-system-defaults/install
-	$(INSTALL_DIR) $(1)/etc/uci-defaults
-	$(CP) ./uci-defaults/* $(1)/etc/uci-defaults
-endef
+include ../../freifunk-berlin-generic.mk
 
-$(eval $(call BuildPackage,freifunk-berlin-system-defaults))
+# call BuildPackage - is done via freifunk-berlin-generic.mk --> luci.mk
diff --git a/defaults/freifunk-berlin-uhttpd-defaults/Makefile b/defaults/freifunk-berlin-uhttpd-defaults/Makefile
index df98843b1511e6c4ce722dcee501e7ce06624232..97305dad66adf66b6a8db9ef93617d20db221eea 100644
--- a/defaults/freifunk-berlin-uhttpd-defaults/Makefile
+++ b/defaults/freifunk-berlin-uhttpd-defaults/Makefile
@@ -1,35 +1,10 @@
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=freifunk-berlin-uhttpd-defaults
-
-include ../../freifunk-berlin-generic.mk
-
-define Package/freifunk-berlin-uhttpd-defaults
-  SECTION:=freifunk-berlin
-  CATEGORY:=freifunk-berlin
-  TITLE:=Freifunk Berlin uhttpd default configuration
+LUCI_TITLE:=Freifunk Berlin uhttpd default configuration
   URL:=http://github.com/freifunk-berlin/packages_berlin
   DEPENDS+= +uhttpd +freifunk-berlin-lib-guard
-  PKGARCH:=all
-endef
-
-define Package/freifunk-berlin-uhttpd-defaults/description
-  Freifunk Berlin configuration files for uhttpd
-endef
+LUCI_DESCRIPTION=Freifunk Berlin configuration files for uhttpd
 
-define Build/Prepare
-	mkdir -p $(PKG_BUILD_DIR)
-endef
-
-define Build/Configure
-endef
-
-define Build/Compile
-endef
-
-define Package/freifunk-berlin-uhttpd-defaults/install
-	$(INSTALL_DIR) $(1)/etc/uci-defaults
-	$(CP) ./uci-defaults/* $(1)/etc/uci-defaults
-endef
+include ../../freifunk-berlin-generic.mk
 
-$(eval $(call BuildPackage,freifunk-berlin-uhttpd-defaults))
+# call BuildPackage - is done via freifunk-berlin-generic.mk --> luci.mk
diff --git a/defaults/freifunk-berlin-uhttpd-defaults/root/etc/uci-defaults/freifunk-berlin-uhttpd-defaults b/defaults/freifunk-berlin-uhttpd-defaults/root/etc/uci-defaults/freifunk-berlin-uhttpd-defaults
new file mode 100644
index 0000000000000000000000000000000000000000..8428893fc519bce190731cc785aad1ba66fa72bc
--- /dev/null
+++ b/defaults/freifunk-berlin-uhttpd-defaults/root/etc/uci-defaults/freifunk-berlin-uhttpd-defaults
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+. /lib/functions/guard.sh
+guard "uhttpd"
+
+# insert a random part into commonname (Freifunk Berlin - <random>)
+# this should help us to avoid different certificates with same
+# commonname/issuer id
+uci set uhttpd.px5g.commonname="Freifunk Berlin - $(dd if=/dev/urandom bs=4 count=1 | hexdump -e '1/1 "%02x"')"
+# do not force redirect to https
+uci set uhttpd.main.redirect_https=0
+uci commit
diff --git a/defaults/freifunk-berlin-uhttpd-defaults/uci-defaults/freifunk-berlin-uhttpd-defaults b/defaults/freifunk-berlin-uhttpd-defaults/uci-defaults/freifunk-berlin-uhttpd-defaults
deleted file mode 100644
index 8428893fc519bce190731cc785aad1ba66fa72bc..0000000000000000000000000000000000000000
--- a/defaults/freifunk-berlin-uhttpd-defaults/uci-defaults/freifunk-berlin-uhttpd-defaults
+++ /dev/null
@@ -1,12 +0,0 @@
-#!/bin/sh
-
-. /lib/functions/guard.sh
-guard "uhttpd"
-
-# insert a random part into commonname (Freifunk Berlin - <random>)
-# this should help us to avoid different certificates with same
-# commonname/issuer id
-uci set uhttpd.px5g.commonname="Freifunk Berlin - $(dd if=/dev/urandom bs=4 count=1 | hexdump -e '1/1 "%02x"')"
-# do not force redirect to https
-uci set uhttpd.main.redirect_https=0
-uci commit
diff --git a/freifunk-berlin-generic.mk b/freifunk-berlin-generic.mk
index 82d5b602561d9ed3bf229f9f27b6e354eddf9f04..4f55def0635e61a5b7b6da6704aa6344fd1bf1ad 100644
--- a/freifunk-berlin-generic.mk
+++ b/freifunk-berlin-generic.mk
@@ -15,7 +15,10 @@ else
 #$(info found luci.mk at $(LUCIMKFILE))
 endif
 
-PKG_VERSION?=$(if $(DUMP),x,$(strip $(shell \
+LUCI_SECTION?=freifunk-berlin
+LUCI_CATEGORY?=freifunk-berlin
+
+xx__PKG_VERSION?=$(if $(DUMP),x,$(strip $(shell \
 	if svn info >/dev/null 2>/dev/null; then \
 		revision="svn-r$$(LC_ALL=C svn info | sed -ne 's/^Revision: //p')"; \
 	elif git log -1 >/dev/null 2>/dev/null; then \
diff --git a/uplinks/freifunk-berlin-uplink-notunnel/Makefile b/uplinks/freifunk-berlin-uplink-notunnel/Makefile
index 2d128f70b09681aa7316e557389c1f17a97631fa..2565cd06d8ccb313d895275bf4fa0b3d4f937ca3 100644
--- a/uplinks/freifunk-berlin-uplink-notunnel/Makefile
+++ b/uplinks/freifunk-berlin-uplink-notunnel/Makefile
@@ -1,43 +1,14 @@
 
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=freifunk-berlin-uplink-notunnel-files
-
-include ../../freifunk-berlin-generic.mk
-
-define Package/$(PKG_NAME)
-  SECTION:=freifunk-berlin
-  CATEGORY:=freifunk-berlin
-  TITLE:=Freifunk Berlin no tunnel files
+LUCI_TITLE:=Freifunk Berlin no tunnel files
   URL:=http://github.com/freifunk-berlin/packages_berlin
   DEPENDS+=+freifunk-berlin-lib-guard +kmod-veth +freifunk-berlin-network-defaults +pingcheck
   PROVIDES:=freifunk-berlin-uplink
-  PKGARCH:=all
-endef
 
-define Package/$(PKG_NAME)/description
-  Freifunk Berlin files to setup a uplink without any
+LUCI_DESCRIPTION=Freifunk Berlin files to setup a uplink without any
   tunnel.
   upstream traffic will be directly routed via your network
-endef
-
-define Build/Prepare
-	mkdir -p $(PKG_BUILD_DIR)
-endef
 
-define Build/Configure
-endef
-
-define Build/Compile
-endef
-
-define Package/$(PKG_NAME)/install
-	$(INSTALL_DIR) $(1)/etc/uci-defaults
-	$(CP) ./uci-defaults/* $(1)/etc/uci-defaults
-	$(INSTALL_DIR) $(1)/etc/pingcheck/online.d
-	$(CP) ./files/etc/pingcheck/online.d/* $(1)/etc/pingcheck/online.d
-	$(INSTALL_DIR) $(1)/etc/pingcheck/offline.d
-	$(CP) ./files/etc/pingcheck/offline.d/* $(1)/etc/pingcheck/offline.d
-endef
+include ../../freifunk-berlin-generic.mk
 
-$(eval $(call BuildPackage,$(PKG_NAME)))
