From: Sven Roederer <freifunk@it-solutions.geroedel.de>
Date: Tue, 21 Apr 2020 02:06:03 +0200
Subject: migration: do not run on development systems

diff --git a/utils/freifunk-berlin-migration/uci-defaults/freifunk-berlin-01-migration.sh b/utils/freifunk-berlin-migration/uci-defaults/freifunk-berlin-01-migration.sh
index ac725d0a901e8033d3f0a074292fa2519def9fb6..2d0b24a2714589cc3e79957a7e5d73825298bd9c 100755
--- a/utils/freifunk-berlin-migration/uci-defaults/freifunk-berlin-01-migration.sh
+++ b/utils/freifunk-berlin-migration/uci-defaults/freifunk-berlin-01-migration.sh
@@ -5,6 +5,11 @@ source /lib/functions/semver.sh
 source /etc/openwrt_release
 source /lib/functions/guard.sh
 
+if [ $( uci -q get system.@system[0].was_development) = "1" ]; then
+  echo "Non released code was ran on system. skipping migration ..."
+  exit 0
+fi
+
 # possible cases: 
 # 1) firstboot with kathleen --> uci system.version not defined
 # 2) upgrade from kathleen --> uci system.version defined
