From: Sven Roederer <devel-sven@geroedel.de>
Date: Sat, 17 Apr 2021 22:40:18 +0200
Subject: freifunk-generic.mk: allow overwriting auf all LUCI_* variables

diff --git a/freifunk-berlin-generic.mk b/freifunk-berlin-generic.mk
index ff24284edc6ba85007dd7f2a7255fe7ca8e2b677..ab38403e63542cc70b8d1efdb297deaee41dcc20 100644
--- a/freifunk-berlin-generic.mk
+++ b/freifunk-berlin-generic.mk
@@ -15,10 +15,11 @@ else
 #$(info found luci.mk at $(LUCIMKFILE))
 endif
 
-LUCI_SECTION=freifunk-berlin
-LUCI_CATEGORY=freifunk-berlin
-LUCI_MAINTAINER=Freifunk Berlin community
-LUCI_URL=http://github.com/freifunk-berlin/packages_berlin
+LUCI_SECTION?=freifunk-berlin
+LUCI_CATEGORY?=freifunk-berlin
+LUCI_MAINTAINER?=Freifunk Berlin community
+LUCI_URL?=http://github.com/freifunk-berlin/packages_berlin
+LUCI_SUBMENU?=
 
 PKG_VERSION?=$(if $(DUMP),x,$(strip $(shell \
 	if svn info >/dev/null 2>/dev/null; then \
