From: Sven Roederer <devel-sven@geroedel.de>
Date: Sat, 12 Jun 2021 23:27:05 +0200
Subject: network-defaults: br-sertup: fix

diff --git a/defaults/freifunk-berlin-network-defaults/root/etc/uci-defaults/freifunk-berlin-network-defaults b/defaults/freifunk-berlin-network-defaults/root/etc/uci-defaults/freifunk-berlin-network-defaults
index 590028ef9093b1dd9782852dbcf1405339b20a99..bdf66b482fbae751cc624ae22143ab7d20f1f734 100644
--- a/defaults/freifunk-berlin-network-defaults/root/etc/uci-defaults/freifunk-berlin-network-defaults
+++ b/defaults/freifunk-berlin-network-defaults/root/etc/uci-defaults/freifunk-berlin-network-defaults
@@ -24,6 +24,7 @@ wan_2_bridge() {
   WANDEVICESECTION=$(config_foreach find_first_section_for_name device network ${WANDEVICE})
   # define WAN-device as bridge with name "br-wan"
   uci set network.$WANDEVICESECTION.type='bridge'
+  uci add_list network.$WANDEVICESECTION.ports=$(uci get network.$WANDEVICESECTION.name)
   uci set network.$WANDEVICESECTION.name='br-wan'
   # set WAN & WAN6 interfaces to new WAN-device
   uci set network.wan.device="br-wan"
