From: Sven Roederer <devel-sven@geroedel.de>
Date: Fri, 11 Jun 2021 00:53:32 +0200
Subject: wizard: only setup redirection when wizard has never run

this prevents setup of redirection on every firmware-upgrade

diff --git a/utils/luci-app-ffwizard-berlin/root/etc/uci-defaults/wizarddefaults b/utils/luci-app-ffwizard-berlin/root/etc/uci-defaults/wizarddefaults
index 086f9dfd60deb77f61b5c3ed5ce9d24737737799..b0714f097c58a7af2875f4d6a31c019fe927cdff 100644
--- a/utils/luci-app-ffwizard-berlin/root/etc/uci-defaults/wizarddefaults
+++ b/utils/luci-app-ffwizard-berlin/root/etc/uci-defaults/wizarddefaults
@@ -7,6 +7,8 @@ uci set freifunk-policyrouting.pr.fallback=0
 uci set freifunk-policyrouting.pr.zones="freifunk"
 uci commit freifunk-policyrouting
 
+# redirect to wizard when it has never run before
+[ $(uci -q get ffwizard.settings.runbefore) = 'true' ] && exit 0
 uci set freifunk.luci=setting
 uci set freifunk.luci.redirect_landingurl='/cgi-bin/luci/admin/freifunk/assistent?luci_username=root&luci_password='
 uci commit freifunk.luci
