From: Sven Roederer <devel-sven@geroedel.de>
Date: Sun, 6 Jun 2021 16:32:29 +0200
Subject: wizard: changes to logging in wireless (verbose and changed)

diff --git a/utils/luci-app-ffwizard-berlin/luasrc/model/cbi/freifunk/assistent/wireless.lua b/utils/luci-app-ffwizard-berlin/luasrc/model/cbi/freifunk/assistent/wireless.lua
index bd12a855681cdf0928f1b3e366d2d2fc1a65404b..645c82ac6fce73e8197b1b0eb9b3d35848f0b2ab 100644
--- a/utils/luci-app-ffwizard-berlin/luasrc/model/cbi/freifunk/assistent/wireless.lua
+++ b/utils/luci-app-ffwizard-berlin/luasrc/model/cbi/freifunk/assistent/wireless.lua
@@ -299,7 +299,11 @@ function main.write(self, section, value)
   uci:save("dhcp")
   uci:save("olsrd")
   uci:save("olsrd6")
-  uci:save("network")
+  if uci:save("network") == true then
+    tools.logger("uci:save network passed", "ffwizard",5)
+  else
+    tools.logger("uci:save network failed", "ffwizard",5)
+  end
   uci:save("ffwizard")
 end
 
