From: Sven Roederer <devel-sven@geroedel.de>
Date: Mon, 19 Apr 2021 01:42:10 +0200
Subject: adapt to LUCI_MENU

diff --git a/freifunk-berlin-generic.mk b/freifunk-berlin-generic.mk
index ab38403e63542cc70b8d1efdb297deaee41dcc20..5d749f243baec15e77d7fd67a981897509c4a959 100644
--- a/freifunk-berlin-generic.mk
+++ b/freifunk-berlin-generic.mk
@@ -19,7 +19,7 @@ LUCI_SECTION?=freifunk-berlin
 LUCI_CATEGORY?=freifunk-berlin
 LUCI_MAINTAINER?=Freifunk Berlin community
 LUCI_URL?=http://github.com/freifunk-berlin/packages_berlin
-LUCI_SUBMENU?=
+LUCI_MENU?=none
 
 PKG_VERSION?=$(if $(DUMP),x,$(strip $(shell \
 	if svn info >/dev/null 2>/dev/null; then \
diff --git a/utils/luci-mod-freifunk-ui/Makefile b/utils/luci-mod-freifunk-ui/Makefile
index 4dcdbd87471ad5fcfa37053bcaf86e9165883174..45ab0a6d20b04edb0dfb80cb16f57d6fcdef4c56 100644
--- a/utils/luci-mod-freifunk-ui/Makefile
+++ b/utils/luci-mod-freifunk-ui/Makefile
@@ -10,7 +10,7 @@ include $(TOPDIR)/rules.mk
 LUCI_TITLE:=Freifunk Public and Admin LuCI UI
 LUCI_SECTION:=luci
 LUCI_CATEGORY:=LuCI
-LUCI_SUBMENU:=2. Modules
+LUCI_MENU:=2. Modules
 LUCI_DEPENDS:=+luci-mod-admin-full +luci-lib-json +community-profiles +luci-lib-ipkg
 LUCI_PKGARCH:=all
 PKG_RELEASE:=3
