From: Sven Roederer <freifunk@it-solutions.geroedel.de>
Date: Sat, 5 Jun 2021 04:12:30 +0200
Subject: wizard: create br-dhcp in new uci-config

diff --git a/utils/luci-app-ffwizard-berlin/luasrc/model/cbi/freifunk/assistent/wireless.lua b/utils/luci-app-ffwizard-berlin/luasrc/model/cbi/freifunk/assistent/wireless.lua
index 3ead64f810431e51d9d604a8669e12ce3bd13819..bd57fc39569336df5eac8b486a02bbb5293d4f23 100644
--- a/utils/luci-app-ffwizard-berlin/luasrc/model/cbi/freifunk/assistent/wireless.lua
+++ b/utils/luci-app-ffwizard-berlin/luasrc/model/cbi/freifunk/assistent/wireless.lua
@@ -239,16 +239,34 @@ function main.write(self, section, value)
   --only do this if user entered cidr
   if (dhcpmeshnet:prefix() < 32) then
     --NETWORK CONFIG bridge for wifi APs
+    -- rename device br-lan to br-dhcp
+    uci:foreach("network","device",
+      function(section)
+--        print(section.name)
+        if section.name == "br-lan" then
+--          print("--match--")
+--      print('------------------')
+--      for key, value in pairs(section) do
+--        print(key .. ': ' .. tostring(value))
+--      end
+--      print(section[0])
+--      print(uci:set("network", section['.name'], "name", "br-dhcp"))
+          uci:set("network", section['.name'], "name", "br-dhcp")
+        end
+      end
+    )
+
     local prenetconfig =  {}
     prenetconfig.ipaddr=dhcpmeshnet:minhost():string()
     prenetconfig.netmask=dhcpmeshnet:mask():string()
     prenetconfig.ip6assign="64"
-    prenetconfig.type="bridge"
+--    prenetconfig.type="bridge"
     prenetconfig.proto="static"
     -- use ifname from dhcp bridge on a consecutive run of assistent
-    prenetconfig.ifname=uci:get("network", "lan", "ifname") or uci:get("network", "dhcp", "ifname")
+    prenetconfig.ifname="br-dhcp"
 
     -- if macaddr is set for lan interface also set it for dhcp interface (needed for wdr4900)
+    -- TODO: rewite to new network --> is it needed? the MAC should belong to the still correct device
     local macaddr=uci:get("network", "lan", "macaddr") or uci:get("network", "dhcp", "macaddr")
     if (macaddr) then
       prenetconfig.macaddr = macaddr
