From: Sven Roederer <devel-sven@geroedel.de>
Date: Sun, 6 Jun 2021 16:32:29 +0200
Subject: wizard: changes to logging in wireless (verbose and changed)

diff --git a/utils/luci-app-ffwizard-berlin/luasrc/model/cbi/freifunk/assistent/wireless.lua b/utils/luci-app-ffwizard-berlin/luasrc/model/cbi/freifunk/assistent/wireless.lua
index be4aef3f1bc0d70cda52fe1a6eff45aa557e6d1d..135955a6e9c44dd7c4c6e652bb98622e1c0516fc 100644
--- a/utils/luci-app-ffwizard-berlin/luasrc/model/cbi/freifunk/assistent/wireless.lua
+++ b/utils/luci-app-ffwizard-berlin/luasrc/model/cbi/freifunk/assistent/wireless.lua
@@ -243,17 +243,27 @@ function main.write(self, section, value)
     -- rename device br-lan to br-dhcp
     uci:foreach("network","device",
       function(section)
---        print(section.name)
+        tools.logger("found a section: " .. tostring(section), "ffwizard-devicesection",5)
         if section.name == "br-lan" then
-          tools.logger("found br-lan device","ffwizard",5)
+          tools.logger("found device br-lan","ffwizard",5)
           tools.logger('dumping contens',"ffwizard",5)
           tools.dumptable(section)
---      print(section[0])
---      print(uci:set("network", section['.name'], "name", "br-dhcp"))
-          tools.logger(tostring(uci:set("network", section['.name'], "name", "br-dhcp")),"ffwizard",5)
+          local result= uci:set("network", section['.name'], "name", "br-dhcp")
+          if result == true then
+            tools.logger('uci set was successful',"ffwizard",5)
+          else
+            tools.logger('uci set failed',"ffwizard",5)
+          end
         end
       end
     )
+    tools.logger("dumping device-section after change","ffwizard",5)
+    uci:foreach("network","device",
+      function(section)
+        tools.logger('dumping contens of section ' .. tostring(section),"ffwizard",5)
+        tools.dumptable(section)
+      end
+    )
 
     local prenetconfig =  {}
     prenetconfig.ipaddr=dhcpmeshnet:minhost():string()
@@ -314,7 +324,11 @@ function main.write(self, section, value)
   uci:save("dhcp")
   uci:save("olsrd")
   uci:save("olsrd6")
-  uci:save("network")
+  if uci:save("network") == true then
+    tools.logger("uci:save network passed", "ffwizard",5)
+  else
+    tools.logger("uci:save network failed", "ffwizard",5)
+  end
   uci:save("ffwizard")
 end
 
